docs/docs/genai/eval-monitor/running-evaluation/agents.mdx

```diff
@@ -119,25 +119,28 @@ def predict_fn(question: str) -> str:
 
 ### Step 2: Create evaluation dataset
 
-Design test cases as a list of dictionaries, each with an `inputs` and `expectations` field. We would like to evaluate the correctness of the output, but also the tool calls used by the agent.
+Design test cases as a list of dictionaries, each with an `inputs`, `expectations`, and `tags` field. We would like to evaluate the correctness of the output, but also the tool calls used by the agent.
```

Comment:
might want to mention that tags are optional. As it reads right now it sounds like this is mandatory for the input data structure.